### PR TITLE
Astro 4.7

### PR DESCRIPTION
**Pull request by dmcalester**
It’s Astro Launch Tuesday™

Please review for launch today. Major checks are:

- [Flight Dynamics](http://draft--astrouxds.netlify.com/fd-service-ux-design/about-the-fd-designs/)
- [Compliance](http://draft--astrouxds.netlify.com/design-guidelines/compliance/) bumped to 1.2 for Drop Down/Select
- [Content Policy](http://draft--astrouxds.netlify.com/community/content-policy/)
- [Select Menu](http://draft--astrouxds.netlify.com/components/select/) rename

### CODE REVIEW

File: _content/_data/navigation.json [hunk 1]
```diff
@@ -261,6 +260,25 @@
...
+	{
+		"label": "Community",
+		"url": "/community",
+		"items": [
+			{
+				"label": "Propose a Change",
+				"url": "/community/propose-a-change"
+			},
+			{
+				"label": "Content Policy",
```
Comment (sknep) [inline _content/_data/navigation.json:277]: So the content policy is going live?

File: _content/patterns/classification-markings.md [hunk 2]
```diff
@@ -1,262 +0,0 @@
----
```
Comment (sknep) [inline _content/patterns/classification-markings.md:1]: Was this file published in error? Or was it once live? If it was once live, we need to make sure there's a redirect set up for this URL in netlify, per the content policy https://draft--astrouxds.netlify.app/community/content-policy/#4.1.-deleting-entire-pages

File: _content/downloads.md [hunk 3]
```diff
@@ -10,17 +10,19 @@ title: Downloads
 
 ## Astro Design System
 
-- Astro Component Source Code ([Git Repository](https://bitbucket.org/rocketcom/astro-components/src/master/))
+- Astro Component Source Code ([Git Repository](https://github.com/RocketCommunicationsInc/astro-components))
```
Comment (sknep) [inline _content/downloads.md:13]: we need to update the following bitbucket URLs soon!

File: netlify.toml [hunk 4]
```diff
@@ -68,4 +68,10 @@
 	from = "/tt-c-service-ux-design/tt-c-investigate"
 	to = "/ttc-service-ux-design/ttc-investigate"
 	status = 300
+	force = true
+
+	[[redirects]]
+	from = "/components/drop-down"
```
Comment (sknep) [inline netlify.toml:74]: Thanks!

File: _content/_data/navigation.json (hunk 1 above)
Comment (dmcalester) [reply _content/_data/navigation.json:277]: Yes

File: _content/patterns/classification-markings.md (hunk 2 above)
Comment (dmcalester) [reply _content/patterns/classification-markings.md:1]: Yes it was. Thank you.

**Pull request merged**